### ISSUE TITLE
Mixing brush modes

### ISSUE THREAD

**Issue by julianheinrich** [syntagmatic/parallel-coordinates#275]
Currently, lines can be selected via a single brush mode. It would be really useful though to be able to iteratively select lines using angular brushing, then strums, etc.

I guess this can be worked around by keeping track of brushed lines, but having this functionality in the library would also allow to keep the brushes on the plot.

**Comment by antz29** [comment]
:+1:

**Comment by bbroeksema** [comment]
That would not be very straightforward to implement. I have implemented brushing  more-or-less as a mutual-exclusive plugin system. Now, it might be possible to rewrite this a bit to enable two plugins at the same time but I'm not confident that there will be no unwanted interplay between two brush implementations as all of them listen to mouse events. I wouldn't be surprised if you'd encounter unwanted side-effects.

**Comment by julianheinrich** [comment]
Yes, I wasn' t expecting this to be straight forward. Would it be possible to listen to events based on the current mouse position? For example, allow a 1D brush to be created/modified only when hovering over axes and 2D only between axes.
